File: Python Exams/Basic Exam Collection/Basics Online Exam - 15 and 16 June 2019/02. Lunch Break/02. Lunch Break.py
from math import ceil

serial_name = input()
chapter_time = int(input())
rest_time = int(input())

lunch_time = rest_time * 1/8
other_rest_time = rest_time * 1/4
time_left = rest_time - lunch_time - other_rest_time
total_time = chapter_time - time_left

if time_left >= chapter_time:
    print(f'You have enough time to watch {serial_name} and left with {ceil(abs(total_time))} minutes free time.')
else:
    print(f'You don\'t have enough time to watch {serial_name}, you need {ceil(total_time)} more minutes.')

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 15 and 16 June 2019/02. Lunch Break/02. Lunch Break.js
function lunchBreak(serialName, chapterTime, restTime) {
    let lunchTime = restTime * 1 / 8
    let otherRestTime = restTime * 1 / 4
    let timeLeft = restTime - lunchTime - otherRestTime
    let totalTime = chapterTime - timeLeft

    if (timeLeft >= chapterTime)
        return `You have enough time to watch ${serialName} and left with ${Math.ceil(Math.abs(totalTime))} minutes free time.`
    return `You don\'t have enough time to watch ${serialName}, you need ${Math.ceil(totalTime)} more minutes.`
}

// console.log(lunchBreak('Game of Thrones', 60, 96))
// console.log(lunchBreak('Teen Wolf', 48, 60))